ci/deploy_mz/pipeline.yml

```diff
@@ -8,6 +8,14 @@
 # by the Apache License, Version 2.0.
 
 steps:
+  - command: bin/ci-builder run stable bin/pyactivate -m ci.deploy_mz.version
+    timeout_in_minutes: 30
+    concurrency: 1
+    concurrency_group: deploy-mz/version
+    retry:
+      manual:
+        permit_on_passed: true
```

Comment:
@def-: This sounds very interesting. Is this something we could use in the coverage analysis to allow retries after having re-run some build steps?

Reply:
Yes, good catch, I'll put up a PR to allow this in coverage runs.